src/Listener/UpdateCastorListener.php

```diff
@@ -46,7 +46,7 @@ public function checkUpdate(ConsoleCommandEvent $event): void
         if (!$command) {
             return;
         }
-        if ('_complete' === $command->getName()) {
+        if ('completion' === $command->getName()) {
```

Comment:
Both command should be checked then (`completion` is the autocomplete installation script and `_complete` is the command internally called to provide suggestions according to the current input)